diff --git a/static/vendors/mathbox/mathbox-bundle.js b/static/vendors/mathbox/mathbox-bundle.js
index 9f9fcf4..70eeded 100644
--- a/static/vendors/mathbox/mathbox-bundle.js
+++ b/static/vendors/mathbox/mathbox-bundle.js
@@ -42814,6 +42814,9 @@ THREE.Bootstrap.registerPlugin('time', {
 
       api.frames++;
 
+      // Display fps
+      $("#fps").text(api.fps.toString().substring(0, 5));
+
       if (this.wait-- > 0) {
         this.clockStart = clock;
         this.timeStart  = time;
diff --git a/templates/index.html b/templates/index.html
index 679f66d..e7e8fb8 100644
--- a/templates/index.html
+++ b/templates/index.html
@@ -71,6 +71,7 @@
           <a class="navbar-brand" href="#">Math3d</a>
         </div>
         <ul class="nav navbar-nav navbar-right">
+          <li>FPS: <span id="fps">0</span></li>
           <li ng-controller="popoverCtrl"
               popover-is-open="myPopover.isOpen"  
               popover-class="popover-settings"
